src/execution/logging.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+export const TAG = {
+  SUBTREE_START: 'SUBTREE_START',
```

Comment:
Isn't a `SUBTREE` just a non-leaf node? I don't understand what the reason to log here is since the `RESOLVER` will log immediately after. Seems redundant but I'm probably missing something.